lib/bitcoin/IProofOfFeeConfig.ts

```diff
@@ -0,0 +1,22 @@
+
+/**
+ * Defines configuration of approximate sliding windows used in
+ * proof of fee calculation.
+ */
+export interface ISlidingWindowQuantileConfig {
+  windowSizeInBatches: number;
```

Comment:
Can we find a different word term than `batch` 'coz every time I see it, I think of Sidetree operation batches, which I think is entirely different here. Perhaps `aggregation` or `grouping` or `set`?

Reply:
went with groupId

Reply:
I went with 'group'.